### ISSUE TITLE
plantUML ParticipantPadding and BoxPadding skinparams not working

### ISSUE THREAD

**Issue by pedro-girao** [asciidoctor/asciidoctor-diagram#152]
Hello!

I think the title says it all. 
In other words, given the sequence diagram example provided in the PlanUML page:

```
@startuml
skinparam ParticipantPadding 20
skinparam BoxPadding 10

box "Foo1"
participant Alice1
participant Alice2
end box
box "Foo2"
participant Bob1
participant Bob2
end box
Alice1 -> Bob1 : hello
Alice1 -> Out : out
@enduml

```
Using asciidoctor diagram the boxes are generated with the same size and always at the same distance, whatever the values given to ParticipantPadding and BoxPadding. Even tried to put them in a config.txt file and use it, but it yelds the same result. 

Are these skinparams not supported? Or is it some other issue?

Thank you!
Pedro.

**Comment by pepijnve** [comment]
Everything in the block is passed on to PlantUML as is, so there's no inherent reason why these would not work. Perhaps these were introduced in a more recent version of PlantUML than the one that's included in asciidoctor-diagram?

**Comment by pedro-girao** [comment]
Yes you're right! I replaced the plantuml.jar with the most recent one and it worked just fine.
Should it be updated with the new jar?

**Comment by pepijnve** [comment]
This was fixed in 1.5.5
